<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>689</width>
    <height>449</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>689</width>
    <height>449</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>689</width>
    <height>449</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>Courier New</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>Omron_pyModbus TCP/IP</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>11</y>
     <width>41</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Courier New</family>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>IP</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>43</y>
     <width>51</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Courier New</family>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Port</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="ipTxt">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>7</y>
     <width>151</width>
     <height>23</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="inputMask">
    <string/>
   </property>
   <property name="text">
    <string>192.168.250.0</string>
   </property>
   <property name="maxLength">
    <number>16</number>
   </property>
   <property name="cursorMoveStyle">
    <enum>Qt::LogicalMoveStyle</enum>
   </property>
  </widget>
  <widget class="QLineEdit" name="portTxt">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>40</y>
     <width>71</width>
     <height>23</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="inputMethodHints">
    <set>Qt::ImhNone</set>
   </property>
   <property name="inputMask">
    <string/>
   </property>
   <property name="text">
    <string>502</string>
   </property>
   <property name="maxLength">
    <number>32767</number>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Normal</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="openBtn">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>80</y>
     <width>101</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Open</string>
   </property>
  </widget>
  <widget class="QPushButton" name="closeBtn">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>80</y>
     <width>101</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Close</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="rwDregister">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>140</y>
     <width>221</width>
     <height>121</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="title">
    <string>R/W D Register</string>
   </property>
   <widget class="QPushButton" name="dReadBtn">
    <property name="geometry">
     <rect>
      <x>13</x>
      <y>90</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Read</string>
    </property>
   </widget>
   <widget class="QPushButton" name="dWriteBtn">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>90</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Write</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>13</x>
      <y>30</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <underline>true</underline>
     </font>
    </property>
    <property name="text">
     <string>Addr:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="dAddrTxt">
    <property name="geometry">
     <rect>
      <x>13</x>
      <y>50</y>
      <width>71</width>
      <height>23</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>30</y>
      <width>61</width>
      <height>13</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <underline>true</underline>
     </font>
    </property>
    <property name="text">
     <string>Value:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="dValTxt">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>50</y>
      <width>71</width>
      <height>23</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="rwWregister">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>310</y>
     <width>221</width>
     <height>121</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="title">
    <string>R/W W Register</string>
   </property>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>13</x>
      <y>30</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
      <underline>true</underline>
     </font>
    </property>
    <property name="text">
     <string>Addr:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="wAddrTxt">
    <property name="geometry">
     <rect>
      <x>13</x>
      <y>50</y>
      <width>51</width>
      <height>23</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>30</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <underline>true</underline>
     </font>
    </property>
    <property name="text">
     <string>Bit:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="wBitTxt">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>50</y>
      <width>51</width>
      <height>23</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="wReadBtn">
    <property name="geometry">
     <rect>
      <x>13</x>
      <y>90</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Read</string>
    </property>
   </widget>
   <widget class="QPushButton" name="wWriteBtn">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>90</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Write</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>30</y>
      <width>61</width>
      <height>13</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <underline>true</underline>
     </font>
    </property>
    <property name="text">
     <string>Value:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="wValTxt">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>50</y>
      <width>51</width>
      <height>23</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QTextEdit" name="outputTxt">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>40</y>
     <width>401</width>
     <height>391</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="mouseTracking">
    <bool>false</bool>
   </property>
   <property name="inputMethodHints">
    <set>Qt::ImhMultiLine</set>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>11</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Courier New</family>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Response</string>
   </property>
  </widget>
  <widget class="QPushButton" name="clearBtn">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>10</y>
     <width>101</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Clear</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
